contracts/utils/Ownable.sol

```diff
@@ -18,6 +20,8 @@ abstract contract Ownable {
         _;
     }
 
+    /// @notice Activate new ownership
+    /// @notice Called by nominated owner only
```

Comment:
nit:
```suggestion
    /// @notice Only nominated owner can call
```

Reply:
Adjusted in 4fed1e9c109a6478d2a92f53649d77877b9f9668